src/SpecificDissipationRateEquationSystem.C

```diff
@@ -242,7 +246,8 @@ SpecificDissipationRateEquationSystem::register_interior_algorithm(
     if (itsi == solverAlgDriver_->solverAlgMap_.end()) {
       SolverAlgorithm* theAlg = NULL;
       if (realm_.realmUsesEdges_) {
-        theAlg = new ScalarEdgeSolverAlg(realm_, part, this, sdr_, dwdx_, evisc_);
+        const bool useAvgMdot = (realm_.solutionOptions_->turbulenceModel_ == SST_TAMS) ? true : false;
+        theAlg = new ScalarEdgeSolverAlg(realm_, part, this, sdr_, dwdx_, evisc_, useAvgMdot);
```

Comment:
Does the average mass flow rate satisfy continuity? Do you experience any issues with this?

Reply:
Good question. We think so and haven't had any issues with it. Given that's it's just a linear combination of mass flow rates, we think this isn't a problem. But we haven't formally verified it.